src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/GEOS_LandAssimGridComp.F90

```diff
@@ -1594,9 +1588,9 @@ subroutine RUN ( GC, IMPORT, EXPORT, CLOCK, RC )
           read(datestamp(10:13),*) nhms
           nhms = nhms*100
           do ens = 0, NUM_ENSEMBLE-1
-             call get_id_string(id_string, ens + FIRST_ENS_ID, ens_id_width)
+             call get_ensid_string(ensid_string, ens + FIRST_ENS_ID, ens_id_width) !xxxx, not _exxxx
```

Comment:
I don't understand the comment here:   `...  ! xxxx not _exxxx`
Doesn't get_ensid_string produce "_eXXXX"?
[Same comment applies to line ~1268 above.]